src/CodeGeneration.jl

```diff
@@ -19,16 +20,42 @@ generatestruct(name::Symbol, supertype::Symbol, fields::Vararg{Tuple{Symbol, Sym
         $(map(x -> :($(x[1])::$(x[2])), fields)...) 
     end)
 
+# Generate import
+
+generateimport(mod::Expr, ::Vararg{Symbol,0}) = :(import $mod)
+generateimport(mod::Expr, objs::Vararg{Symbol,N}) where {N} = :(import $mod; $(objs...))
```

Comment:
Are we importing from a module? If so,
```suggestion
generateimport(mod::Expr, objs::Vararg{Symbol,N}) where {N} = :(import $mod: $(objs...))
```

Reply:
I was trying something but then it became irrelevant. However, we can keep it there, it might be useful.